src/test/enhetstester/kotlin/no/nav/familie/ks/sak/kjerne/personopplysninggrunnlag/PersonopplysningGrunnlagServiceTest.kt

```diff
@@ -38,6 +39,9 @@ internal class PersonopplysningGrunnlagServiceTest {
     @MockK
     private lateinit var arbeidsfordelingService: ArbeidsfordelingService
 
+    @MockK
+    private lateinit var personidentService: PersonidentService
```

Comment:
Så at denne ikke ble brukt, men det er fordi det ikke finnes tester for den nye `oppdaterPersonopplysningGrunnlag` akkurat nå.

Reply:
Ah, det hadde jeg helt glemt! Skal få lagt det inn! 👍